api/src/routers/reviews.js

```diff
@@ -0,0 +1,131 @@
+import express from "express";
+import knex from "../database_client.js";
+import asyncHandler from "./error_handler.js";
+
+const reviewsRouter = express.Router();
+
+reviewsRouter.get(
+  "/",
+  asyncHandler(async (req, res) => {
+    const result = await knex("Review").orderBy("id");
+    res.status(200).json(result);
+  })
+);
+
+reviewsRouter.get(
+  "/:id",
+  asyncHandler(async (req, res) => {
+    const id = req.params.id;
+    const result = await knex("Review").where({ id }).first();
+
+    if (!result)
+      return res.status(404).json({ error: `Review with id ${id} not found` });
+
+    res.status(200).json(result);
+  })
+);
+
+function validateReviewData(data) {
+  const errors = [];
+  if (
+    data.title !== undefined &&
```

Comment:
Usually in JavaScript (and only JavaScript) you can simplify this part as `if (!data.title)`, because the undefined value is treated as false in JavaScript.

This is because sometimes, depending on the context, browser etc, the value might be null instead of undefined, and it is best to guard against both cases (null and undefined).

See https://developer.mozilla.org/en-US/docs/Glossary/Falsy

In other languages like C#, Java etc, the way you are doing it would be the way :)